kitsune/wiki/templates/wiki/review_revision.html

```diff
@@ -20,6 +20,15 @@
         <p>
           {{ _('This revision was already reviewed by {user} on {date}.')|fe(user=revision.reviewer, date=datetimeformat(revision.reviewed, format='longdatetime')) }}
         </p>
+      {% elif document.current_revision > revision %}
```

Comment:
I think you should be more explicit with these comparisons. I think `document.current_revision.created > revision.created` would read much better.